rotkehlchen/chain/evm/decoding/curve_lend/utils.py

```diff
@@ -0,0 +1,133 @@
+import logging
+from json import JSONDecodeError
+from typing import TYPE_CHECKING, Any
+
+import requests
+
+from rotkehlchen.assets.asset import UnderlyingToken
+from rotkehlchen.assets.utils import TokenEncounterInfo, get_or_create_evm_token
+from rotkehlchen.chain.evm.types import string_to_evm_address
+from rotkehlchen.constants import ONE
+from rotkehlchen.db.settings import CachedSettings
+from rotkehlchen.errors.serialization import DeserializationError
+from rotkehlchen.globaldb.cache import (
+    globaldb_get_unique_cache_value,
+    globaldb_set_unique_cache_value,
+)
+from rotkehlchen.globaldb.handler import GlobalDBHandler
+from rotkehlchen.logging import RotkehlchenLogsAdapter
+from rotkehlchen.serialization.deserialize import deserialize_int
+from rotkehlchen.types import (
+    CURVE_VAULTS_PROTOCOL,
+    CacheType,
+    ChainID,
+    EvmTokenKind,
+)
+
+from .constants import CURVE_VAULT_API
+
+if TYPE_CHECKING:
+    from rotkehlchen.db.dbhandler import DBHandler
+
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
+
+
+def _update_cache_timestamp(count: int | None = None) -> None:
+    with GlobalDBHandler().conn.write_ctx() as write_cursor:
+        globaldb_set_unique_cache_value(
+            write_cursor=write_cursor,
+            key_parts=[CacheType.CURVE_VAULTS],
+            value=str(count) if count is not None else '0',
+        )
+
+
+def _query_curve_vaults_api() -> list[dict[str, Any]] | None:
+    """Query Curve vaults from the api.
+    Returns vault list or None if there was an error."""
+    try:
+        response_data = requests.get(
+            url=CURVE_VAULT_API,
```

Comment:
This is the only place this is used in production, right? So hard code it instead.